python/triton/runtime/autotuner.py

```diff
@@ -172,10 +172,13 @@ def check_disk_cache(self, tuning_key, configs, bench_fn):
         from triton._C.libtriton import get_cache_invalidating_env_vars
         from triton.compiler.compiler import make_backend, triton_key
         from triton.runtime.cache import get_cache_manager
+        from triton.runtime.interpreter import InterpretedFunction
         from triton.runtime.jit import JITFunction
 
         fn = self.fn
         while not isinstance(fn, JITFunction):
+            if isinstance(fn, InterpretedFunction):
+                return bench_fn()
```

Comment:
Should it be the following?

```
bench_fn()
return
```

Reply:
`benchmark` also returns `None` so they are equivalent

Reply:
ah yeah for some reason I was just inclined to save a line of code 😅